packages/eslint-plugin/src/rules/no-angle-bracket-type-assertion.ts

```diff
@@ -15,6 +15,8 @@ export default util.createRule({
         "Prefer 'as {{cast}}' instead of '<{{cast}}>' when doing type assertions.",
     },
     schema: [],
+    deprecated: true,
```

Comment:
As we applying this as a breaking change should we just remove the rules? How long were you planning on keeping the deprecated implementations for?

Reply:
Just saw #732 which answers that question!

Let's just get it all done in this PR now that we are into v2 prereleases